engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql

```diff
@@ -539,4 +539,4 @@ CREATE TABLE `cloud`.`template_ovf_properties` (
 
 -- Add VM snapshot ID on usage helper tables
 ALTER TABLE `cloud_usage`.`usage_vmsnapshot` ADD COLUMN `vm_snapshot_id` BIGINT(20) NULL DEFAULT NULL AFTER `processed`;
-ALTER TABLE `cloud_usage`.`usage_snapshot_on_primary` ADD COLUMN `vm_snapshot_id` BIGINT(20) NULL DEFAULT NULL AFTER `deleted`;
+ALTER TABLE `cloud_usage`.`usage_snapshot_on_primary` ADD COLUMN `vm_snapshot_id` BIGINT(20) NULL DEFAULT NULL AFTER `deleted`;
```

Comment:
I believe you should not be touching an old upgrade schema whatsoever?

Reply:
I changed/add nothing to this schema.. Just moved the added lines from Gerd Müller (see annoted ..) to the actual schema.